huaweicloud/services/acceptance/waf/data_source_huaweicloud_waf_certificate_test.go

```diff
@@ -0,0 +1,56 @@
+package waf
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+)
+
+/*
+Before testing, the certificate should be created in SCM and pushed to WAF
+The name of the certificate is: certificate_1,
+or change the value of `name` to the name of your certificate.
+*/
+func TestAccDataSourceWafCertificateV1_basic(t *testing.T) {
+	name := "certificate_1"
```

Comment:
The acc test can not be executed automatically, please add the pre-condition in the Config.

Reply:
fixes the `testAccWafCertificateListV1_conf`